sklearn/feature_selection/univariate_selection.py

```diff
@@ -596,7 +596,7 @@ def _get_support_mask(self):
         n_features = len(self.pvalues_)
         sv = np.sort(self.pvalues_)
         selected = sv[sv <= float(self.alpha) / n_features
-                      * np.arange(n_features)]
+                      * (np.arange(n_features) + 1)]
```

Comment:
fwiw, i think pep8 has changed their stance on this, see [1,2]. It does seem to be what's causing the travis failure though.

[1] https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator
[2] http://bugs.python.org/issue26763